packages/mobile/src/utils/translationsFetcher.ts

```diff
@@ -0,0 +1,11 @@
+import i18n from 'src/i18n'
+
+export const fetchI18nContent = (content: { [lang: string]: any }) => {
+  const language = i18n.language.toLowerCase()
+  const texts = content[language] || content[language.slice(0, 2)] || content.en
+  for (const key of Object.keys(texts)) {
+    // This is needed for newlines to show properly.
+    texts[key] = texts[key].replace(/\\n/g, '\n')
+  }
```

Comment:
nit: I'm not entirely convinced this method should do this for all content.

This is because the Firebase console strips new lines.
If we wrote the value programmatically, this wouldn't happen.

But fine for now.

Reply:
What do you mean programatically? I couldn't find a way for newlines to print correctly without this, but if this isn't necessary I'm happy to remove it :)

Reply:
No practical alternative if we're using the Firebase console to write those fields.

What I mean is, if we were writing to the DB using the Firebase JS SDK for instance we wouldn't need to represent a new line char with a literal `\\n`.
i.e. we would do something like `db.write("somField", "Line 1\nLine2")` and the client would read and display the new line.

Reply:
Oh, I see. That would be nice, I guess we could build a small internal webpage that interacts with Firebase and allows to edit strings from there. It would enable non-technical people to edit those in a more user friendly way.